src/test.rs

```diff
@@ -49,6 +50,30 @@ type PersonAndOpaqueSender = (Person, OpaqueIpcSender);
 type PersonAndReceiver = (Person, IpcReceiver<Person>);
 type PersonAndSharedMemory = (Person, IpcSharedMemory);
 
+#[derive(Clone, Debug, Eq, PartialEq)]
+struct HasWeirdSerializer (Option<String>);
+
+lazy_static! { static ref WEIRD_CHANNEL: (IpcSender<HasWeirdSerializer>, IpcReceiver<HasWeirdSerializer>) = ipc::channel().unwrap(); }
```

Comment:
A non-reentrant testcase for reentrance? No that's just lazy! ;-)

I guess it's not really a problem, unless the test harness ever grows an ability to run the same test multiple times in parallel, which seems rather hypothetical...